packages/paypal-commerce-utils/src/paypal-commerce-sdk.ts

```diff
@@ -90,7 +112,36 @@ export default class PayPalCommerceSdk {
             },
             attributes: {
                 'data-client-metadata-id': 'sandbox', // TODO: should be updated when paypal will be ready for production
-                'data-namespace': 'paypalAxo',
+                'data-namespace': PayPalCommerceSdkNamespaces.PaypalAxo,
+                'data-partner-attribution-id': attributionId,
+                'data-user-id-token': clientToken,
+            },
+        };
+    }
+
+    private getPayPalSdkMessagesConfiguration(
+        paymentMethod: PaymentMethod<PayPalCommerceInitializationData>,
+        currencyCode: string,
+    ): PayPalSdkConfig {
+        const { clientToken, initializationData } = paymentMethod;
+
+        if (!initializationData || !initializationData.clientId) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        const { intent, clientId, merchantId, attributionId } = initializationData;
+
+        return {
+            options: {
+                'client-id': clientId,
+                'merchant-id': merchantId,
+                commit: true,
```

Comment:
I don't really sure that `commit` param should be true for messages... As far as I know, it should be false if we use banners outside checkout page. I might be better to check paypal documentation and update commit param with right value for banners